src/frontend/context/DraftObservationContext.tsx

```diff
@@ -66,32 +65,37 @@ const usePersistedState = createPersistedState(
   }
 );
 
-export type DraftObservationContextType = UseState<DraftObservationContextState>;
+export type DraftObservationContextType = [
+  DraftObservationContextState,
+  React.Dispatch<React.SetStateAction<DraftObservationContextState>>
+];
 
 const defaultContext: DraftObservationContextType = [
   {
     photos: [],
     value: null,
     photoPromises: [],
   },
-  () => {},
+  //@ts-ignore
+  null,
 ];
 
-const DraftObservationContext = React.createContext<DraftObservationContextType>(
-  defaultContext
-);
+const DraftObservationContext = React.createContext<
+  DraftObservationContextType
+>(defaultContext);
 
 type Props = {
-  children: React.Node,
+  children: React.ReactNode;
 };
 
 export const DraftObservationProvider = ({ children }: Props) => {
   const [
     state,
     status,
     setState,
+    // @ts-ignore We know the first item in the array is correct
   ] = usePersistedState<DraftObservationContextState>(defaultContext[0]);
-  const contextValue = React.useMemo(() => [state, setState], [
+  const contextValue = React.useMemo(() => [state, setState] as const, [
```

Comment:
nit, but could avoid the `as const` if you make `DraftObservationContextType` readonly i.e. `type DraftObservationContextType = readonly ...`. not super important though